src/Illuminate/Foundation/Console/PackageDiscoverCommand.php

```diff
@@ -38,7 +38,7 @@ public function handle(PackageManifest $manifest)
 
         (new Collection($manifest->manifest))
             ->keys()
-            ->each(fn ($description) => $this->components->task($description))
+            ->each($this->components->task(...))
```

Comment:
This looks like it could be a breaking change?

`components->task` will now receive the `$description` and also the `$index` as the second parameter, where it is expecting a callable.

Reply:
Confirmed this is a breaking change.

<img width="905" alt="Screenshot 2024-12-10 at 09 19 31" src="https://github.com/user-attachments/assets/df28512e-aefd-4497-9e4c-602e0b74824b">

Reply:
Whoops, totally missed that!